Description: disable GLuint64 on GL ES
Origin: upstream, changeset 3f0065d77ee3
Last-Update: 2013-11-19

--- a/sip/QtGui/opengl_types.sip
+++ b/sip/QtGui/opengl_types.sip
@@ -29,7 +29,9 @@
 typedef quint32 GLuint;
 typedef quint32 GLenum;
 typedef quint32 GLbitfield;
+%If (PyQt_Desktop_OpenGL)
 typedef quint64 GLuint64;
+%End
 typedef float GLfloat;
 typedef float GLclampf;
 typedef double GLdouble;
--- a/sip/QtGui/qpyopengl_collections.sip
+++ b/sip/QtGui/qpyopengl_collections.sip
@@ -19,6 +19,8 @@
 // WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
 
 
+%If (PyQt_Desktop_OpenGL)
+
 // QVector<GLuint64> is implemented as a Python list of integers.
 %MappedType QVector<GLuint64> /DocType="list-of-int"/
 {
@@ -69,3 +71,5 @@
     return sipGetState(sipTransferObj);
 %End
 };
+
+%End
