Description: add qmake variables when generating all .pro files
Origin: upstream, changeset 44c92d7c6f19
Last-Update: 2013-10-21

--- a/configure.py
+++ b/configure.py
@@ -1331,6 +1331,8 @@
 VPATH = %s
 ''' % (qmake_quote(sp_plugin_dir), qmake_quote(sp_plugin_dir)))
 
+    fout.write('\n'.join(target_config.qmake_variables) + '\n')
+
     fout.close()
 
     # Create the makefile.
@@ -1373,6 +1375,7 @@
         pro_lines.append('VPATH = %s' % qmake_quote(sp_src_dir))
 
     pro_lines.extend(pro_sources(sp_src_dir))
+    pro_lines.extend(target_config.qmake_variables)
 
     pro_name = os.path.join(src_dir, app + '.pro')
 
